api/go.mod

```diff
@@ -8,32 +8,29 @@ require (
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/imdario/mergo v0.3.6
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.8.0
+	github.com/stretchr/testify v1.8.1
```

Comment:
Is this bump related somehow? Usually we want each dep in its own PR, but in this case I don't mind too much because testify is a tests-only dep.